Dockerfile

```diff
@@ -1,11 +1,11 @@
 FROM openjdk:8-jre
 
-COPY build/install/tribunals-case-api /opt/app/
+COPY build/libs/tribunals-case-api.jar /opt/app/
 
 WORKDIR /opt/app
 
-HEALTHCHECK --interval=10s --timeout=10s --retries=10 CMD http_proxy="" curl --silent --fail http://localhost:8083/health
+HEALTHCHECK NONE
 
-EXPOSE 8083
+EXPOSE 8082
```

Comment:
sscs case loader uses port 8082. So it will be better to use TCA API default port of 8080

Reply:
Port updated to use 8080